modules/Core/pages/panel/emails.php

```diff
@@ -38,167 +38,244 @@
 $page_title = $language->get('admin', 'emails');
 require_once(ROOT_PATH . '/core/templates/backend_init.php');
 
-if(isset($_GET['action']) && $_GET['action'] == 'test'){
-	$smarty->assign(array(
-		'SEND_TEST_EMAIL' => $language->get('admin', 'send_test_email'),
-		'BACK' => $language->get('general', 'back'),
-		'BACK_LINK' => URL::build('/panel/core/emails')
-	));
+// Since emails are sent in the user's language, they need to be able to pick which language's messages to edit
+$editing_language = new Language(null, Session::get('editing_language') ?? $queries->getWhere('languages', array('is_default', '=', 1))[0]->name);
+$emails = array(
+	['register', 'Registration', ['subject' => $editing_language->get('emails', 'register_subject'), 'message' => $editing_language->get('emails', 'register_message')]],
```

Comment:
Are the English terms (eg `Registration`, `Change Password`) on these lines shown to the user? If so, should they be translated to display in the user's language?

Reply:
Ah, that was silly. Thanks